Sources/Snowplow/Emitter/EmitterEvent.swift

```diff
@@ -18,7 +18,7 @@ public class EmitterEvent: NSObject {
     private(set) var payload: Payload
     private(set) var storeId: Int64
 
-    init(payload: Payload, storeId: Int64) {
+    public init(payload: Payload, storeId: Int64) {
```

Comment:
Also make it available to ObjC.

```suggestion
    @objc
    public init(payload: Payload, storeId: Int64) {
```